share/spack/qa/run-shell-tests

```diff
@@ -18,7 +18,11 @@
 ORIGINAL_PATH="$PATH"
 
 . "$(dirname $0)/setup.sh"
-check_dependencies $coverage kcov git hg svn
+if [ "$COVERAGE" = true ]; then
+  check_dependencies $coverage kcov git hg svn
+else
+  echo "ONLY PACKAGE RECIPES CHANGED [skipping coverage]"
```

Comment:
I think we do not really know here why coverage is being skipped -- maybe just output "COVERAGE not set to 'true'. skipping coverage"